eclass/ros-catkin.eclass

```diff
@@ -163,16 +163,19 @@ ros-catkin_src_prepare() {
 	append-cxxflags '-std=c++11'
 }
 
+ROSPREFIX="/opt/ros/${ROSDISTRO}"
```

Comment:
What's the problem? You can't have a working ros now. (If yes tell me how)
I can remove the `ROSDISTRO` variable